data/data_download/gadm_3.6/Makefile

```diff
@@ -0,0 +1,35 @@
+.PHONY: import
+MAKEFLAGS := --jobs=$(shell nproc)
+MAKEFLAGS += --output-sync=target
+
+MarxanUser:=$(shell psql -X -A -t "postgresql://$$API_POSTGRES_USER:$$API_POSTGRES_PASSWORD@marxan-postgresql-api:5432/$$API_POSTGRES_DB" -c "select id from users limit 1")
+
+import: data/gadm/gadm36_simp.geojson
+	ogr2ogr -makevalid \
+		-overwrite \
+		-nln admin_regions -nlt PROMOTE_TO_MULTI \
+		-lco GEOMETRY_NAME=the_geom \
+		-f PostgreSQL PG:"dbname=$$GEO_POSTGRES_DB host='marxan-postgresql-geo-api' \
+		port=5432 user=$$GEO_POSTGRES_USER password=$$GEO_POSTGRES_PASSWORD" $< \
+		-sql "select *,'$(MarxanUser)' as created_by from \"$$(basename -s .geojson "$<")\"";
```

Comment:
also, some of the params over here may benefit from using env vars, but depending on how much of this will stay once moving to Airflow, we can keep this as is for the moment, and remove hardcoded values later on if still useful/if it makes sense.

Reply:
Yes, we can see further in the development which ones need to be passed as parameters and which ones can stay hardcoded :+1: